pkg/ccl/backupccl/backup_test.go

```diff
@@ -2103,11 +2107,10 @@ func TestBackupRestoreUserDefinedTypes(t *testing.T) {
 SET experimental_enable_enums = true;
 
 CREATE DATABASE d;
+CREATE TYPE d.unused AS ENUM ('lonely');
```

Comment:
still need to test that this type is actually included

Reply:
Woops, thanks for catching. Done.